scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -442,39 +455,39 @@ public void onClick(ClickEvent event) {
             }
         });
 
-        schedStartButton = new MenuItem("Start");
+        schedStartButton = new MenuItem(START);
         schedStartButton.setIcon(SchedulerImages.instance.scheduler_start_16().getSafeUri().asString());
         schedStartButton.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
```

Comment:
I am agree with sonar. IDEA easily can replace anonymous to lambda. (i guess Eclipse will do this as well)

Reply:
![capture](https://user-images.githubusercontent.com/13152450/46013460-a80bb100-c0cc-11e8-8629-63f7fe2e4b93.PNG)

tried this but it breaks all action buttons

Reply:
What do you mean by 'it breaks'? The method they are supposed to call is not called after the refactoring?

Reply:
It means that button are not clickable.. Marco says because i lose the onclick call after the lambda refactoring..